infra/enrolment-records-store/src/main/java/com/simprints/infra/enrolment/records/store/EnrolmentRecordRepository.kt

```diff
@@ -1,9 +1,32 @@
 package com.simprints.infra.enrolment.records.store
 
+import com.simprints.core.ExcludedFromGeneratedTestCoverageReports
 import com.simprints.infra.config.store.models.Project
+import com.simprints.infra.enrolment.records.store.domain.models.BiometricDataSource
+import com.simprints.infra.enrolment.records.store.domain.models.FaceIdentity
+import com.simprints.infra.enrolment.records.store.domain.models.FingerprintIdentity
+import com.simprints.infra.enrolment.records.store.domain.models.SubjectQuery
 import com.simprints.infra.enrolment.records.store.local.EnrolmentRecordLocalDataSource
 
+@ExcludedFromGeneratedTestCoverageReports("This is an interface with no logic")
```

Comment:
Technically, the default values could be considered business logic.

Reply:
Do you think it merits a test?

Reply:
I don't think so, just pointing out a technicality 🤓